core/optionsx.h

```diff
@@ -539,6 +539,11 @@
     OPTION_DEFAULT_INTERNAL(uint, opt_mangle, 1,
                             "optimization level on optimizing mangle sequences")
 #endif
+#ifdef AARCH64
+    OPTION_DEFAULT_INTERNAL(bool, build_ldstex, false,
```

Comment:
Why not put unsafe_ in the name, which is our convention (xref the many other options such as unsafe_ignore_eflags_ibl, etc.)